deploy/docker-compose/server-bare.yml

```diff
@@ -11,8 +11,9 @@ services:
       MYSQL_ROOT_PASSWORD: "sota-test"
       MYSQL_USER: "sota"
       MYSQL_PASSWORD: "s0ta"
-      MYSQL_DATABASES: "sota_resolver sota_resolver_test sota_core sota_core_test"
-
+      MYSQL_DATABASES: "sota_resolver sota_resolver_test sota_resolver% sota_core sota_core_test sota_core% sota_device_registry"
+    command: ["--character-set-server=utf8", "--collation-server=utf8_unicode_ci", "--max-connections=1000"]
```

Comment:
I did verify that those flags are set correctly in the mariadb container, and just now I ran the random tests locally on my machine, and they ran fine.

That said, I didn't re-create the random tests task on teamcity using the new docker-compose file.